### PR TITLE
Fix/change email cherry

### PR DESCRIPTION
**Pull request by Tolik9924**

### CODE REVIEW

File: back-end/services/UserService.js [hunk 1]
```diff
@@ -1,5 +1,9 @@
 const User = require('../models/User');
 
+const getUserById = async (id) => {
```
Comment (andriydzhrvts) [inline back-end/services/UserService.js]: Where you use this method?

File: back-end/controllers/UserController.js [hunk 2]
```diff
@@ -17,8 +18,18 @@ const getUser = async (req, res) => {
...
+      const body = {
+        fullName: req.body.fullName,
+        dateOfBirth: req.body.dateOfBirth,
+      };
+      await updateByID(id, body);
+      return res.status(400).json({ message: 'exist email' });
+    }
     const user = await updateByID(id, req.body);
-    res.status(200).json(user);
+    const token = jwt.sign({ email: user.email }, process.env.SECRET_KEY, { expiresIn: '2h' });
```
Comment (andriydzhrvts) [inline back-end/controllers/UserController.js]: We have the same code in `authService` when we do login, so you can create a method with this logic.

File: back-end/services/UserService.js (hunk 1 above)
Comment (Tolik9924) [reply back-end/services/UserService.js]: I don't use this method but I thought this method could help me.

File: back-end/services/AuthService.js [hunk 3]
```diff
@@ -48,4 +48,8 @@ const hashPassword = (pass) =>
     });
   });
 
-module.exports = { register, login, comparePasswords, hashPassword };
+const createToken = (email) => {
+  return jwt.sign({ email }, process.env.SECRET_KEY, { expiresIn: '2h' });
```
Comment (andriydzhrvts) [inline back-end/services/AuthService.js]: 30d

File: back-end/controllers/UserController.js [hunk 4]
```diff
@@ -23,6 +23,15 @@ const getUser = async (req, res) => {
 const updateUserById = async (req, res) => {
   try {
     const { id } = req.params;
+    const existEmail = await findByEmail(req.body.email);
+    if (existEmail) {
+      const body = {
+        fullName: req.body.fullName,
+        dateOfBirth: req.body.dateOfBirth,
+      };
+      await updateByID(id, body);
```
Comment (andriydzhrvts) [inline back-end/controllers/UserController.js]: Are you sure that when we have existEmail we need to do update user?

Comment (Tolik9924) [reply back-end/controllers/UserController.js]: I thought should update what pass validation

File: back-end/services/UserService.js (hunk 1 above)
Comment (andriydzhrvts) [reply back-end/services/UserService.js]: How this method could help you?)

File: back-end/controllers/UserController.js (hunk 4 above)
Comment (andriydzhrvts) [reply back-end/controllers/UserController.js]: Okay. Why you update and return 400 fail status code?

File: back-end/services/UserService.js (hunk 1 above)
Comment (Tolik9924) [reply back-end/services/UserService.js]: I created this func when started to work for task but realised it unnecessary, can't remember why created this func.I will remove this func

**Review by andriydzhrvts — APPROVED** [review]

**Pull request merged**